src/com/janek/basics/math/MathUtil.java

```diff
@@ -58,4 +66,20 @@ public static double doCalculation(double one, String symbol, double two) {
                 throw new IllegalArgumentException("Unsupported symbol");
         }
     }
+
+    public static boolean isEven(int number) {
```

Comment:
javadoc please

Reply:
The description should be:
```
**Checks** if the number is even.
```
and the return statement:
```
@return true if the number is even, otherwise false
```